SpriterDotNet/SpriterParser.cs

```diff
@@ -13,6 +13,7 @@ namespace SpriterDotNet
     public static class SpriterParser
     {
         public static readonly ICollection<ISpriterParser> Parsers = new List<ISpriterParser>();
+        internal static Action<Spriter> InitAction = parser => { Init(parser); };
```

Comment:
I don't feel comfortable exposing this just for the sake of testing a trivial method call. The Init method is public and static so it it the perfect candidate for unit testing. Also, it doesn't depend on complex conditions to be invoked - just a null check. Maybe a better approach would be put the null check in the beginning of the Init method.